app/models/ext_management_system.rb

```diff
@@ -61,6 +61,7 @@ def self.create_from_params(params, endpoints, authentications)
       endpoints.each { |endpoint| ems.assign_nested_endpoint(endpoint) }
       authentications.each { |authentication| ems.assign_nested_authentication(authentication) }
 
+      ems.provider.save! if ems.provider.present?
```

Comment:
does it make sense to add `&& ems.provider.changed?`

Reply:
:+1: yeah good call, done